cilib/enums.py

```diff
@@ -13,14 +13,14 @@
 # Next MAJOR.MINOR
 # This controls whether or not we publish pre-release snaps in our channels.
 # Typically, this is K8S_STABLE_VERSION +1. However, when prepping the next
-# stable release, this will be +2. For example, 1.26 is currently stable and
-# we're working on the 1.27 GA. Set this value to '1.28' sometime between the
-# final RC and GA so we don't get pre-release builds (e.g. 1.27.1-alpha.0) in
-# our 1.27 tracks.
+# stable release, this will be +2. For example, 1.27 is currently stable and
+# we're working on the 1.28 GA. Set this value to '1.29' sometime between the
+# final RC and GA so we don't get pre-release builds (e.g. 1.28.1-alpha.0) in
+# our 1.28 tracks.
 K8S_NEXT_VERSION = "1.28"
```

Comment:
Shouldn't we set this one to `1.29`
```suggestion
K8S_NEXT_VERSION = "1.29"
```

Reply:
wow, i changed all the comments without setting the value.  Ugh -- this is why we review